src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java

```diff
@@ -281,19 +281,18 @@ private void doRetrieve(SCMHeadObserver observer, TaskListener listener, GHRepos
             PullRequestSCMHead head = new PullRequestSCMHead(ghPullRequest);
             final String branchName = head.getName();
             listener.getLogger().format("%n    Checking pull request %s%n", HyperlinkNote.encodeTo(ghPullRequest.getHtmlUrl().toString(), "#" + branchName));
-            // FYI https://developer.github.com/v3/pulls/#response-1
-            Boolean mergeable = ghPullRequest.getMergeable();
-            if (!Boolean.TRUE.equals(mergeable)) {
-                listener.getLogger().format("    Not mergeable, skipping%n%n");
-                continue;
-            }
             if (repo.getOwner().equals(ghPullRequest.getHead().getUser())) {
                 listener.getLogger().format("    Submitted from origin repository, skipping%n%n");
                 continue;
             }
             if (criteria != null) {
                 SCMSourceCriteria.Probe probe = getProbe(branchName, "pull request", "refs/pull/" + head.getNumber() + "/head", repo, listener);
                 if (criteria.isHead(probe, listener)) {
+                    // FYI https://developer.github.com/v3/pulls/#response-1
+                    Boolean mergeable = ghPullRequest.getMergeable();
```

Comment:
I am not sure what `MARKER_FILE` is, but anyway my comment is just to repeat the bug found by @PerilousApricot: if `getMergeable()` returns null, we have no information about whether the PR is mergeable or not, so we should err on the side of being quiet and not alarming the user with what is probably a false warning.